lib/service/service.go

```diff
@@ -6694,6 +6695,21 @@ func makeXForwardedForMiddleware(cfg *servicecfg.Config) utils.HTTPMiddleware {
 	return utils.NoopHTTPMiddleware
 }
 
+// makeApplicationCORS converts a servicecfg.CORS to a types.CORS.
+func makeApplicationCORS(c *servicecfg.CORS) *types.CORS {
+	if c == nil {
```

Comment:
Does Teleport infers any default rules as of today or it forwards the CORS requests directly into the application?

Reply:
Currently any preflight requests are completely thrown out because they are unauthenticated. So no defaults.